api/v1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
let's add deprecated marker to v1alpha1 (commenting here since those files are not changed)